### PR TITLE
CGP-144: Artemis Proposal to Build Public Goods Data Dashboards for Celo Ecosystem and Community

### PR DESCRIPTION
**Pull request by anthonyyim**

### CODE REVIEW

File: CGPs/cgp-0143.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
+---
+cgp: 143
+title: Artemis Proposal to Build Public Goods Data Dashboards for Celo Ecosystem and Community
+date-created: 2024-08-06
+author: 'Anthony Yim (@andromedae88)'
+status: DRAFT
+discussions-to: https://forum.link - <URL of discussion forum>
```
Comment (work-0xj4an) [inline CGPs/cgp-0143.md:7]: Fix this, are truncated

File: CGPs/cgp-0143.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+## Overview of Artemis Dashboard Proposal
+
+Dear Celo Community,
+
+We are thrilled to propose funding for the development of the Application Activity Monitor and the Stabila Stablecoin Dashboards by Artemis, an institutional data provider trusted by both financial institutions and by various Celo Foundation members. By providing high quality and in-depth dashboards on what dapps are doing on Celo and tracking Stabila performance, we hope to allow Celo Foundation to make informed decisions about how to significantly grow Celo stablecoin usage. In addition, we hope that our institutional investor user base will more clearly understand all the interesting activity occuring on Celo and invest in the ecosystem.
+
+ 
+### Status
+- DRAFT = Feedback collection
```
Comment (work-0xj4an) [inline CGPs/cgp-0143.md:20]: Remove  "= Feedback collection"

File: CGPs/cgp-0143.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+**Total Funding Request**: $274,000 per year
+
+**Multisig Signers**: Add wallet address here about who will be on the multisig that receives the funding
+
+# Conclusion
+Integrating Artemis into the Celo ecosystem will significantly enhance transparency, usability, and adoption of Celo’s blockchain technology, especially towards liquid token investors and large financial institutions–the core customers of Artemis and potential investors of the Celo ecosystem. Many of these customers are not aware of all the new developments of Celo as they understand Celo as a climate focused chain. Artemis can help provide investors an up-to-date, unbiased and data-backed view on the latest developments of Celo including its transition to an Ethereum L2 so that Celo remains top-of-mind in the investor community.
+
+By providing comprehensive data solutions such as the Application Activity Monitor and Stablecoin Dashboard, Artemis will also empower the Celo community with valuable insights and drive informed decision-making. This partnership aligns with Celo’s mission of financial inclusion and stability, supporting the ecosystem’s growth and long-term success.
+
+## Proposed Changes
```
Comment (work-0xj4an) [inline CGPs/cgp-0143.md:166]: Fill out this

File: CGPs/cgp-0143.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+ 
+Fill out the following template for each transaction in the proposal
+ 
+1. Description of transaction 1
+  - Destination: A human readable description of the address and method being called
+  - Data: A human readable description of the transaction data
+  - Value: How much CELO is being sent, and why?
+ 
+## Verification
```
Comment (work-0xj4an) [inline CGPs/cgp-0143.md:175]: You can take an example from here and leave proposal ID as XXX https://github.com/celo-org/governance/blob/main/CGPs/cgp-0140.md

File: CGPs/cgp-0143.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+ 
+An explanation of how voters can verify that this CGP does what it intends to do. Can be left as “TODO” until the proposal is made. Include things like CLI commands to run and pointers to code.
+ 
+## Risks
```
Comment (work-0xj4an) [inline CGPs/cgp-0143.md:179]: Please include any possible Risk asociated with the proposal.

**Review by work-0xj4an — CHANGES_REQUESTED** [review]

File: CGPs/cgp-0143.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+# Funding Request
+Artemis offers comprehensive solutions with the Application Activity Monitor and Stablecoin Dashboard, priced based on the equivalent cost of hiring 2-3 full-time employees (FTEs) per year to build and maintain similar data infrastructure as well as what other ecosystems are paying for Artemis’ services.
+
+- **Application Activity Monitor**: $150,000 per year
+- **Stablecoin Dashboard**: $124,000 per year
+
+Our pricing model reflects the extensive expertise and ongoing support required to ensure these tools are continuously updated, reliable, and user-friendly.
+
+**Total Funding Request**: $274,000 per year
```
Comment (anthonyyim) [inline CGPs/cgp-0143.md:156]: From @work-0xj4an: update to be in CELO not USD using last 90d price

File: CGPs/cgp-0143.md [hunk 7]
```diff
@@ -0,0 +1,197 @@
+---
+cgp: 143
+title: Artemis Proposal to Build Public Goods Data Dashboards for Celo Ecosystem and Community
+date-created: 2024-08-06
+author: "Anthony Yim (@andromedae88)"
+status: DRAFT
+discussions-to: https://forum.celo.org/t/draft-artemis-proposal-to-build-public-goods-data-dashboards-for-celo-ecosystem-and-community/8384
+governance-proposal-id: XXX
```
Comment (work-0xj4an) [inline CGPs/cgp-0143.md:8]: Need to leave in blank

File: CGPs/cgp-0143.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+        "0xd1a06a4477b9E4633E6E56D8f9EB9ccD39cE36Bc",
+        "634994000000000000000000"
+      ],
+      "value": "0"
+    }
+]
+```
+ 
+## Verification
+This command should show a proposal with no transactions:
```
Comment (work-0xj4an) [inline CGPs/cgp-0143.md:189]: It has a transaction, please modifiy the text for: _"A human readable version of this proposal can be found using the following command (on-chain ID determined after submission):"_

**Review by work-0xj4an — CHANGES_REQUESTED** [review]

File: CGPs/cgp-0143.md (hunk 8 above)
Comment (work-0xj4an) [reply CGPs/cgp-0143.md:240]: @anthonyyim please modify text above

File: CGPs/cgp-0143.md (hunk 6 above)
Comment (work-0xj4an) [reply CGPs/cgp-0143.md:156]: @anthonyyim I saw here you asking for $274,000 don't know if they are Celo tokens or USD, please fix for a number in Celo and also label the number as Celo Tokens. 
I saw in the json file you are asking for 634,994 Celo Tokens. If this is the right number please fix it in this paragraph.

**Review by work-0xj4an — CHANGES_REQUESTED** [review]

**Comment by anthonyyim** [comment]
Note to self: update CPG number to be 144 (there exists one for 143 already)

File: CGPs/cgp-0143.md (hunk 8 above)
Comment (anthonyyim) [reply CGPs/cgp-0143.md]: Updated--let me know if it looks right @work-0xj4an

File: CGPs/cgp-0143.md (hunk 6 above)
Comment (anthonyyim) [reply CGPs/cgp-0143.md]: Yep, will update--just finalizing the final proposal price with my team before making the edit. Thanks for your patience!

Comment (work-0xj4an) [reply CGPs/cgp-0143.md]: @anthonyyim let me know when done

File: CGPs/cgp-0144.md [hunk 9]
```diff
@@ -0,0 +1,253 @@
...
+
+# Funding Request
+Artemis offers comprehensive solutions with the Application Activity Monitor and Stablecoin Dashboard, priced based on the equivalent cost of hiring 2-3 full-time employees (FTEs) per year to build and maintain similar data infrastructure as well as what other ecosystems are paying for Artemis’ services.
+
+- **Application Activity Monitor**: $150,000 per year
+- **Stablecoin Dashboard**: $124,000 per year
+
+Our pricing model reflects the extensive expertise and ongoing support required to ensure these tools are continuously updated, reliable, and user-friendly.
+
+~~Total Funding Request: US$274,000 per year~~
```
Comment (work-0xj4an) [inline CGPs/cgp-0144.md:156]: Please update numbers to match actual request.

**Review by work-0xj4an — CHANGES_REQUESTED** [review]

**Review by work-0xj4an — APPROVED** [review]

**Review by 0xzoz — APPROVED** [review]

**Review by work-0xj4an — APPROVED** [review]

**Pull request merged**